dotcom-rendering/src/web/layouts/InteractiveImmersive.stories.tsx

```diff
@@ -40,6 +45,11 @@ const convertToInteractiveImmersive = (CAPI: CAPIType) => {
 const HydratedLayout = ({ ServerCAPI }: { ServerCAPI: CAPIType }) => {
 	fireAndResetHydrationState();
 	const NAV = extractNAV(ServerCAPI.nav);
+	const format: ArticleFormat = {
+		display: decideDisplay(ServerCAPI.format),
+		design: decideDesign(ServerCAPI.format),
+		theme: decideTheme(ServerCAPI.format),
+	};
```

Comment:
It really feels like there should be a single `decideFormat` function that takes care of this?

Reply:
I very much had the same thought when I was writing this code!